docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+= Troubleshooting Landrush
```

Comment:
So, I go along with @budhrg here. If this is a general troubleshooting guide it should be "Troubleshooting" and the intro should talk about troubleshooting in general. Landrush becomes then a second level heading. Let's take height for what we want to do.